_content/excerpts/memento-mori_analayo.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Memento Mori: Recollection of Death in Early Buddhist Meditation"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/memento.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1qSL9gP1zHLPomgBqHo2mkTG6psSG7BlI/view?usp=sharing"
+course: problems
+tags:
+  - death
+  - memory
```

Comment:
🤔 Instead of `memory` and `recollection` let's use `sati` and `tmt` (the tag for "Terror Management Theory")